### PR TITLE
adding the helm values and container image support for the new compli…

### PR DESCRIPTION
**Pull request by janedotx**
…ance api and service

## 📚 Purpose
Adding container images support and helm values for the new compliance packages

## 👌 Resolves:
- [ ] 🐛 JRA-834 fixes bug
- [ ] ✨ JRA-756 implements new feature

## 📦 Impacts:
* mds-compliance-api
* mds-compliance-engine

## ✅ PR Checklist
- [ ] Add or remove checklist items to suit your needs

### CODE REVIEW

File: container-images/mds-compliance-service/webpack.config.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import webpack from '@mds-core/mds-webpack-config'
+
+export default webpack.Bundle('../../packages/mds-jurisdiction-service/server').UsingDefaultConfig()
```
Comment (avatarneil) [inline container-images/mds-compliance-service/webpack.config.ts:3]: Wrong service, should have a path to `../../packages/mds-compliance-service/server`

Comment (janedotx) [reply container-images/mds-compliance-service/webpack.config.ts:3]: opes

File: container-images/mds-compliance-service/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "test": "yarn test:eslint",
+    "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'"
+  },
+  "keywords": [
+    "mds"
+  ],
+  "author": "City of Los Angeles",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@mds-core/mds-jurisdiction-service": "0.1.0",
```
Comment (avatarneil) [inline container-images/mds-compliance-service/package.json:22]: Wrong dependency I think?

Comment (janedotx) [reply container-images/mds-compliance-service/package.json:22]: I could've sworn I'd fixed that...

**Review by avatarneil — APPROVED** [review]

**Pull request merged**